row_batch.go

```diff
@@ -23,6 +24,19 @@ func (e *RowBatch) Values() []RowData {
 	return e.values
 }
 
+func (e *RowBatch) EstimateByteSize() uint64 {
+	var total int
+	for _, v := range e.values {
+		size, err := json.Marshal(v)
```

Comment:
`unsafe.Sizeof` (and `reflect` - they're aliases) give you the size of the pointer. So regardless of string length, the size is 8. Same with uint64, its always gonna be 8. So it's not quite the same since we want to know the byte length of the data itself. 

And with some benchmarking, the `json.Marshal` seems harmless 👇 

```
BenchmarkSize-12         	1000000000	         0.000454 ns/op	       0 B/op	       0 allocs/op     #2.8MB file
BenchmarkJSON-12         	1000000000	         0.00634 ns/op	       0 B/op	       0 allocs/op     #2.8MB file
BenchmarkSizeSmall-12    	1000000000	         0.000077 ns/op	       0 B/op	       0 allocs/op     #200KB file
BenchmarkJSONSmall-12    	1000000000	         0.000429 ns/op	       0 B/op	       0 allocs/op     #200KB file
```

Reply:
Yeah you'd have to traverse into the pointer and things gets ugly quickly (although json.Marshal is technically doing it).

Reply:
Thinking out loud here, but we could technically with minimal performance hit wrap `mysql.writeExecutePacket` and `len` the return data after filtering for `INSERTS`.

Reply:
Or no we can't, it doesn't return the data, just an error.